input/reproSchemaScript/reprotofhirjson.py

```diff
@@ -0,0 +1,282 @@
+'''
+script to convert repro scheme to json for fhir questionnaire
+Not perfect conversion
+#example: "python reprotofhirjson.py <reproschema_folder> valueset"
+must be in the file structure specified in:
+https://github.com/ReproNim/reproschema-library/tree/43e7afab312596708c0ad4dfd45b69c8904088ae/activities
+
+'''
+import json
+import sys
+import os
+import logging
+from dotenv import load_dotenv
+
+load_dotenv()
+CODESYSTEM_URI = os.getenv('CODESYSTEM_URI')
+VALUESET_URI = os.getenv('VALUESET_URI')
+QUESTIONNAIRE_URI = os.getenv('QUESTIONNAIRE_URI')
+
+codeSystem_dict = dict()
+
+fhir_questionnaire = dict()
```

Comment:
having globals like this is a bad habit, because it makes the code less modular and it's hard to follow. instead define this within the function and return it.